java-10/Dockerfile

```diff
@@ -15,11 +15,13 @@ ENV DEFAULT_JVM_OPTS="-XX:+UnlockExperimentalVMOptions"
 ENV MAIN_CLASS="Main"
 ENV CLASSPATH="/app/WEB-INF/classes:/app/WEB-INF/lib/*"
 
-COPY ./run-java.sh /
+COPY init-scripts /init-scripts
+COPY entrypoint.sh /
```

Comment:
Should we explicitly set the execute bit on `entrypoint.sh` and `run-java.sh` in the Dockerfile? If I recall there was a bug with this a while back.

Reply:
The executable bit is set on the files in git, so they should get the same permissions in the container as well.